docs/source/release_notes.rst

```diff
@@ -2,6 +2,7 @@ Release Notes
 -------------
 **Future Releases**
     * Enhancements
+        * Using the ``describe`` method on an un-instantiated ComponentGraph now gives a more helpful error message :pr:`3569`
```

Comment:
Thanks for adding this! Personally, I would consider this change a "fix" instead of an "enhancement", since it's patching an existing bug.

Also, [this is a super nitpick and we haven't been doing a great job of maintaining the pattern recently but] generally we try to keep our changes in past tense (v.0.52.0 is a good example of that). So instead, this commend might be something closer to "Added a clearer error message when ``describe`` is called on an un-instantiated ComponentGraph"